BlackJack.py

```diff
@@ -0,0 +1,235 @@
+from random import choice
+from aenum import Enum, NoAlias
+from UserInputProvider import *
+
+
+class Color(Enum):
+    HEART = 1
+    DIAMOND = 2
+    SPADE = 3
+    CLUB = 4
+
+    @classmethod
+    def values(cls):
+        return list(cls)
+
+
+class Sign(Enum):
+    _settings_ = NoAlias
+    ACE = 1
+    TWO = 2
+    THREE = 3
+    FOUR = 4
+    FIVE = 5
+    SIX = 6
+    SEVEN = 7
+    EIGHT = 8
+    NINE = 9
+    TEN = 10
+    JACK = 10
+    QUEEN = 10
+    KING = 10
+
+    @classmethod
+    def values(cls):
+        return list(cls)
+
+
+class Card(object):
+
+    def __init__(self, sign, color):
+        self.sign = sign
+        self.color = color
+
+    def __repr__(self):
+        return self.sign.name + " " + self.color.name
+
+
+class Deck(object):
+
+    def __init__(self):
+        self.cards = []
+        for color in Color.values():
+            for sign in Sign.values():
+                self.cards.append(Card(sign, color))
+
+    def shuffled_deck(self):
```

Comment:
jest sens to tak robic?
moze lepiej: [https://docs.python.org/3/library/random.html#random.shuffle] 
dodatkowo - shuffle_deck - czasownik, co robi ta funkcja. I warto sie zastanowic, czy nie bedziesz chciala zawsze dostawac potasowanego decku, to znaczy czy nie warto by bylo wrzucic do __init__ tego shufflowania? Możliwe, że nie, wtedy Deck() bedzie bardziej uniwersalny. Pomysl